web/public/mgmt/config.yml

```diff
@@ -135,95 +135,104 @@ collections:
       - { label: "Optional Label", name: "optionalLabel", widget: "string", required: false }
       - { label: "Bolded Heading", name: "headingBolded", widget: "string", required: false }
       - { label: "Sub Heading", name: "headingNotBolded", widget: "string", required: false }
-  - label: "Cannabis License"
-    name: "cannabis-license"
+
+  - name: "cannabis-license"
+    label: "Cannabis License"
     delete: false
     create: false
     files:
-      - label: "Annual - Bottom-of-Task text"
-        name: "annual-bottom-of-task"
-        file: "content/src/display-content/cannabis-license/annual-bottom-of-task.md"
-        identifier_field: "{{fields.slug}}"
-        slug: "{{fields.slug}}"
-        editor:
-          preview: true
-        fields:
-          - { label: "Content", name: "body", widget: "markdown" }
-      - label: "Annual - General Requirements"
-        name: "annual-general-requirements"
-        file: "content/src/display-content/cannabis-license/annual-general-requirements.md"
-        identifier_field: "{{fields.slug}}"
-        slug: "{{fields.slug}}"
-        editor:
-          preview: true
-        fields:
-          - { label: "Content", name: "body", widget: "markdown" }
-      - label: "Conditional - Bottom-of-Task text"
-        name: "conditional-bottom-of-task"
-        file: "content/src/display-content/cannabis-license/conditional-bottom-of-task.md"
-        identifier_field: "{{fields.slug}}"
-        slug: "{{fields.slug}}"
-        editor:
-          preview: true
-        fields:
-          - { label: "Content", name: "body", widget: "markdown" }
-      - label: "Conditional - General Requirements"
-        name: "conditional-general-requirements"
-        file: "content/src/display-content/cannabis-license/conditional-general-requirements.md"
-        identifier_field: "{{fields.slug}}"
-        slug: "{{fields.slug}}"
-        editor:
-          preview: true
-        fields:
-          - { label: "Content", name: "body", widget: "markdown" }
-      - label: "Diversely-Owned Requirements"
-        name: "diversely-owned-requirements"
-        file: "content/src/display-content/cannabis-license/diversely-owned-requirements.md"
-        identifier_field: "{{fields.slug}}"
-        slug: "{{fields.slug}}"
+      - label: Cannabis Eligibility Modal
+        name: cannabis-eligibility-modal
+        file: content/src/fieldConfig/config.json
         editor:
-          preview: true
+          preview: false
         fields:
-          - { label: "Content", name: "body", widget: "markdown" }
-      - label: "Impact-Zone Requirements"
-        name: "impact-zone-requirements"
-        file: "content/src/display-content/cannabis-license/impact-zone-requirements.md"
+          - label: Cannabis Eligibility Modal
```

Comment:
we could make a preview for this, right? just another preview component that just shows the modal?

Reply:
wait no... I remember trying to do this for profile... Material really doesn't let us show the modal in-page instead of opening it across the whole page like a real modal. 

maybe we could fix that at some point but I retract my comment for the purpose of merging this branch